### ISSUE TITLE
Enhancement

### ISSUE THREAD

**Issue by LibertyType** [RafalBuchner/StemThickness#19]
How much trouble would it be to enhance the plugin so when it is alongside a path on the left or right for it to display the number of units to the zero or width?

**Comment by mekkablue** [comment]
I do not understand completely. You mean the distance to LSB and RSB? Can you make a mockup?

**Comment by LibertyType** [comment]
Yes, distance to LSB/RSB. Right now the plugin shows values everywhere the mouse goes except for those two. Making that work would make it a perfect plugin.
